scripts/us_census/acs5yr/subject_tables/s2201/process.py

```diff
@@ -0,0 +1,236 @@
+        'NoWorkersInThePast12Months',
+    '1 worker in past 12 months':
+        'OneWorkerInThePast12Months',
+    '2 or more workers in past 12 months':
+        'TwoOrMoreWorkersInThePast12Months',
+    'Households not receiving food stamps':
+        'WithoutFoodStampsInThePast12Months',
+    'Households not receiving food stamps/SNAP':
+        'WithoutFoodStampsInThePast12Months',
+    'Households not receiving food stamps/SNAP MOE':
+        'WithoutFoodStampsInThePast12Months',
+}
+
+
+def convert_column_to_stat_var(column):
+    """Converts input CSV column name to Statistical Variable DCID."""
+    s = column.split('!!')
+    if 'Median income (dollars)' in s:
+        sv = ['Median_Income_Household']
+    else:
+        sv = ['Count_Household']
+    for p in s:
+
+        # Prefix MOE SVs
+        if p == 'Margin of Error':
+            sv = ['MarginOfError'] + sv
+
+        # Only include the most specific enum value when there are multiple
+        elif p == 'Other family:' and (
+                'Male householder, no spouse present' in s or
+                'Female householder, no spouse present' in s):
+            continue
+        elif p == 'Other family' and (
+                'Male householder, no wife present' in s or
+                'Female householder, no husband present' in s):
+            continue
+        elif p == 'One race' and (
+                'White' in s or 'Black or African American' in s or
+                'American Indian and Alaska Native' in s or 'Asian' in s or
+                'Native Hawaiian and Other Pacific Islander' in s or
+                'Some other race' in s):
+            continue
+
+        elif p in _FEATURE_TO_PROPERTY:
+
+            # Work status is only provided for FamilyHousehold
+            if ((p == 'No workers in past 12 months' or
+                 p == '1 worker in past 12 months' or
+                 p == '2 or more workers in past 12 months') and
+                    'Families' not in s):
+                sv.append(_FEATURE_TO_PROPERTY['Families'])
+            sv.append(_FEATURE_TO_PROPERTY[p])
+    return '_'.join(sv)
+
+
+def create_csv(output, stat_vars):
+    """Creates output CSV file."""
+    fieldnames = ['observationDate', 'observationAbout'] + stat_vars
+    with open(output, 'w') as f_out:
+        writer = csv.DictWriter(f_out, fieldnames=fieldnames)
+        writer.writeheader()
+
+
+def write_csv(filename, reader, output, stat_vars):
+    """Reads input_file and writes cleaned CSV to output."""
+    if 'ACSST5Y' not in filename:
+        return
+    fieldnames = ['observationDate', 'observationAbout'] + stat_vars
+    stat_var_set = set(stat_vars)
+    with open(output, 'a') as f_out:
+        writer = csv.DictWriter(f_out, fieldnames=fieldnames)
+        observation_date = filename.split('ACSST5Y')[1][:4]
+        valid_columns = {}
+        for row in reader:
+            if row['GEO_ID'] == 'id':
+
+                # Map feature names to stat vars
```

Comment:
In the next pass, it might be good to provide this valid_columns map as input to this process.py script. And separate out the convert_column_to_stat_var().

So however we curate the map (handcrafted / automated), we can produce cleaned CSV and TMCF.

Reply:
moved the map as input flag